app/docker/docker-compose.athens.yml

```diff
@@ -0,0 +1,35 @@
+version: '3.5'
+
+services:
+
+  athens:
+    container_name: dab_athens
+    image: "${DAB_APPS_ATHENS_IMAGE:-gomods/athens}:${DAB_APPS_ATHENS_TAG:-latest}"
```

Comment:
When there is a good chance of customizing the config of an app in the past we have used a Dockerfile inside dab and use build args to parameterize the customization (for example have a look at how `DAB_APPS_POSTGRES_LOCALE` env var is handled for the postgres app), is that something that we could do here or are the customizations users are likely to make more private or slow to build such that you want to build centrally and push it to a shared registry?

Reply:
The changes I'm making are specifically to set git config:
```
[url "http://gerrit.lan/"]
        insteadOf = https://gerrit.lan/
        insteadOf = http://gerrit.lan/a/
        insteadOf = git://gerrit.lan/
        insteadOf = ssh://gerrit.lan/
```
so that's fairly bespoke to this company :/

Reply:
Indeed, but simple enough to be doable with a single environment variable that filters through a docker build arg to generate this file, this is very similar then to locale and seems you would want to customize this any time you use athens. For example a build arg with a value of `gerrit.lan` could be used with a few echo's in a run directive or whatever you prefer for templating in a dockerfile something like this:

```Dockerfile
echo "
[url "http://${DAB_APPS_ATHENS_PRIVATE_GIT_FORGE}/"]
        insteadOf = https://${DAB_APPS_ATHENS_PRIVATE_GIT_FORGE}/
        insteadOf = http://${DAB_APPS_ATHENS_PRIVATE_GIT_FORGE}/a/
        insteadOf = git://${DAB_APPS_ATHENS_PRIVATE_GIT_FORGE}
        insteadOf = ssh://${DAB_APPS_ATHENS_PRIVATE_GIT_FORGE}
" > ~/.gitconfig
```